src/unity/python/turicreate/data_structures/sarray_builder.py

```diff
@@ -40,22 +39,19 @@ class SArrayBuilder(object):
     --------
     >>> from turicreate import SArrayBuilder
 
-    >>> sb = SArrayBuilder()
+    >>> sb = SArrayBuilder(int)
 
     >>> sb.append(1)
 
-    >>> sb.append([2,3])
+    >>> sb.append_multiple([2,3])
 
     >>> sb.close()
     dtype: int
     Rows: 3
     [1, 2, 3]
-
     """
     def __init__(self, dtype, num_segments=1, history_size=10):
         self._builder = UnitySArrayBuilderProxy()
-        if dtype is None:
-            dtype = type(None)
```

Comment:
Does `dtype = None` work ?

Reply:
It does not. Error message: `TypeError: dtype argument cannot be None.`